modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISJsonTestSetup.java

```diff
@@ -37,11 +37,21 @@ protected void setUpData() throws Exception {
 
         try (Connection cx = getConnection();
                 Statement st = cx.createStatement();
-                ResultSet rs = st.executeQuery("select Version()")) {
+                ResultSet rs = st.executeQuery("SELECT current_setting('server_version_num')")) {
```

Comment:
Was the old code not working on your setup? Just wondering about this change.

Reply:
Correct, this was failing on my setup.  Selecting version will often return build information text that was not being handled by the GT version comparison tools.  current_setting is the recommended way to get numerical information from Postgres for comparison.